source/fuzz/transformation_add_dead_break.cpp

```diff
@@ -145,22 +145,10 @@ bool TransformationAddDeadBreak::IsApplicable(
     return false;
   }
 
-  assert(bb_from != nullptr &&
-         "We should have found a block if this line of code is reached.");
-  assert(
-      bb_from->id() == message_.from_block() &&
-      "The id of the block we found should match the source id for the break.");
-  assert(bb_to != nullptr &&
-         "We should have found a block if this line of code is reached.");
-  assert(
-      bb_to->id() == message_.to_block() &&
-      "The id of the block we found should match the target id for the break.");
```

Comment:
I removed these since we have already done all the checks for `nullptr` above.

Reply:
I suggest you put these back: an assertion should never fail, so once a piece of code is in good shape all assertions are in some sense redundant; but they are there to defend against mistakes being introduced when code changes.  E.g. what if someone accidentally removed the null check above?  It would be nice for these assertions to fail.

Reply:
Fixed in 2a0b8d1.